docs/source/common/trainer.rst

```diff
@@ -1746,6 +1746,18 @@ The current logger being used. Here's an example using tensorboard
         tensorboard = logger.experiment
 
 
+loggers (p)
+**********
+
+The list of loggers currently being used. Here's an example using tensorboard
+
+.. code-block:: python
+
+    def training_step(self, batch, batch_idx):
+        loggers = self.trainer.loggers
+        tensorboard = loggers[0].experiment
```

Comment:
- we don't need to make this tensorboard specific again
- the example can cite something actually required on the logger base API, like this:
```
loggers = trainer.loggers
for logger in loggers:
    logger.log_metrics({"foo": 1.0})
```

there's no point in mentioning self.trainer.loggers in the lightning module if there's already a loggers property defined directly inside lightning module!